influxdb3/src/commands/serve.rs

```diff
@@ -201,10 +201,10 @@ pub async fn command(config: Config) -> Result<()> {
     info!(%num_threads, "Creating shared query executor");
     let parquet_store =
         ParquetStorage::new(Arc::clone(&object_store), StorageId::from("influxdb3"));
-    let exec = Arc::new(Executor::new_with_config(
-        "datafusion",
+    let mut runtime_builder = tokio::runtime::Builder::new_multi_thread();
+    runtime_builder.worker_threads(num_threads.into());
+    let exec = Arc::new(Executor::new_with_config_and_executor(
```

Comment:
Will have to verify that this is correct, the `iox_query` changed and no longer accepts the thread count as an argument to the `ExecutorConfig`.

Reply:
I updated this to use the `TokioDatafusionConfig` clap block as IOx does, which exposes `datafusion-num-threads` as well as other `datafusion-*` configuration parameters.